base/applications/rapps/appview.cpp

```diff
@@ -1524,7 +1524,7 @@ BOOL CApplicationView::ProcessWindowMessage(HWND hwnd, UINT message, WPARAM wPar
     case WM_NOTIFY:
     {
         LPNMHDR pNotifyHeader = (LPNMHDR)lParam;
-        if (pNotifyHeader->hwndFrom == m_ListView->GetWindow())
+        if (pNotifyHeader->hwndFrom == m_ListView->m_hWnd)
```

Comment:
Note that other parts of the code already access the **public member** `m_hWnd` of the different window objects, so having a `GetWindow()` accessor in the `CUiWindow` seems overkill.

Reply:
I don't like this change, is it just changing code for the sake of changing it?
What actual problem does this solve?